frontend/package-lock.json

```diff
@@ -21961,6 +22256,73 @@
       "resolved": "https://registry.npmjs.org/csstype/-/csstype-3.0.11.tgz",
       "integrity": "sha512-sa6P2wJ+CAbgyy4KFssIb/JNMLxFvKF1pCYCSXS8ZMuqZnMsrxqI2E5sPyoTpxoPU/gVZMzr2zjOfg8GIZOMsw=="
     },
+    "d3-array": {
```

Comment:
Are we using d3 as a dependency?

Reply:
It appears to be in use, uninstalling d3 did nothing.